src/main/java/net/minecraftforge/event/entity/player/PlayerInteractEvent.java

```diff
@@ -292,6 +322,34 @@ public void setCanceled(boolean canceled)
                 useItem = DENY;
             }
         }
+
+        public static enum Action {
+        	/**
+        	 * When the player first left clicks a block
+        	 */
+        	START,
+        	/**
+        	 * When the player stops left clicking a block by completely breaking it
+        	 */
+        	STOP,
+        	/**
+        	 * When the player stops left clicking a block by releasing the button, or no longer targeting the same block
+        	 */
+        	ABORT,
+        	/**
+        	 * When the player is actively mining a block on the client side
```

Comment:
Add the warning message here that it will be called every tick as long as the block is being destroyed.